src/training/tesstrain_utils.py

```diff
@@ -419,7 +419,7 @@ def phase_I_generate_image(ctx, par_factor=None):
 
 # Phase UP : Generate (U)nicharset and (P)roperties file.
 def phase_UP_generate_unicharset(ctx):
-    log.info("=== Phase UP: Generating unicharset and unichar properties files ===")
+    log.info("=== Phase UP: Generating unicharset and unichar properties files from Training Text ===")
 
     box_files = pathlib.Path(ctx.training_dir).glob("*.box")
```

Comment:
`box_files` is now unused and should be removed.

Reply:
Thanks @stweil. I have made the requested change.